tests/deprecated_api/test_remove_1-7.py

```diff
@@ -46,3 +47,25 @@ def test_v1_7_0_deprecated_model_size():
         match="LightningModule.model_size` property was deprecated in v1.5 and will be removed in v1.7"
     ):
         _ = model.model_size
+
+
+def test_v1_7_0_datamodule_transform_properties(tmpdir):
+    dm = MNISTDataModule()
```

Comment:
add a test with a dummy datamodule which passes the transforms & dims constructor arguments to the base DataModule, and confirm that the deprecation warning appears for those as well